dashboard/app/views/devise/sessions/new.html.haml

```diff
@@ -69,3 +70,12 @@
 
     dashboard.clientState.reset();
   });
+
+  function logCreateAccountEvent() {
+    const clickCreateAccountEvent = new CustomEvent('customLogEvent', {
+      detail: {
+        eventName: 'SIGN_UP_STARTED_EVENT'
+      }
+    });
+    document.dispatchEvent(clickCreateAccountEvent);
+  }
```

Comment:
this is a tough one because the haml file already has javascript in it. my request would be to extract this javascript into a new js file, and then add your js to it. you can wrap the existing javascript starting with `  $("#user_login").placeholder();` in a `$(document).ready(` block within the new js file.